### PR TITLE
Update two manifest member (BCD data removed)

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

see https://github.com/mdn/browser-compat-data/pull/23774, the two's BCD data is removed, but the two is still standard: so `browser-compat` key removed and `spec-urls` key added, also update *Browser compatibility* section with a sentence explaining the support status

### CODE REVIEW

File: files/en-us/web/manifest/categories/index.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ The `categories` member is an array of strings defining the names of categories
 
 ## Browser compatibility
 
-{{Compat}}
+The feature is not supported by any of the browser implement yet.
```
Comment (dipikabh) [inline files/en-us/web/manifest/categories/index.md:37]:
```suggestion
The feature is not implemented by any browser yet.
```

File: files/en-us/web/manifest/screenshots/index.md [hunk 2]
```diff
@@ -93,4 +93,4 @@ The `screenshots` member is an array of objects each representing a screenshot.
 
 ## Browser compatibility
 
-{{Compat}}
+The feature is not supported by any of the browser implement yet.
```
Comment (dipikabh) [inline files/en-us/web/manifest/screenshots/index.md:96]: same here

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks a lot @skyclouds2001 for this PR! You beat me to it :)

The updates are good, just a minor nit with the wording

File: files/en-us/web/manifest/screenshots/index.md (hunk 2 above)
Comment (skyclouds2001) [inline files/en-us/web/manifest/screenshots/index.md:96]:
```suggestion
The feature is not implemented by any browser yet.
```

**Comment by Josh-Cena** [comment]
This wording implies that they should be supported by browsers but they are not. While if you read the BCD PR the motivation is "this is an app store feature and has nothing to do with browsers". Therefore we should probably say something similar to that: "Browser compatibility is irrelevant because this member is not used by browsers, but by app stores when publishing your app."

**Comment by skyclouds2001** [comment]
> Browser compatibility is irrelevant because this member is not used by browsers, but by app stores when publishing your app.

I think the following is better:

> Browser compatibility is irrelevant because this feature is not used by browsers, but by app stores when publishing your app. Though some browsers may use this feature, there is no requirement for browsers to implement any form of support."

**Comment by dipikabh** [comment]
Thanks for pointing that out @Josh-Cena, that makes sense.

We could word it as:

> This manifest member is used by app stores and catalogs when publishing and listing web apps, so browser compatibility is not applicable. Browsers may parse this information, but it's optional and doesn't affect the core functionality of a web app. 

@skyclouds2001, WDYT?

File: files/en-us/web/manifest/screenshots/index.md [hunk 3]
```diff
@@ -93,4 +93,4 @@ The `screenshots` member is an array of objects each representing a screenshot.
 
 ## Browser compatibility
 
-{{Compat}}
+The feature is not implemented by any browser yet.
```
Comment (skyclouds2001) [inline files/en-us/web/manifest/screenshots/index.md:96]:
```suggestion
This manifest member is used by app stores and catalogs when publishing and listing web apps, so browser compatibility is not applicable. Browsers may parse this information, but it's optional and doesn't affect the core functionality of a web app.
```

File: files/en-us/web/manifest/categories/index.md [hunk 4]
```diff
@@ -34,4 +34,4 @@ The `categories` member is an array of strings defining the names of categories
 
 ## Browser compatibility
 
-{{Compat}}
+The feature is not implemented by any browser yet.
```
Comment (skyclouds2001) [inline files/en-us/web/manifest/categories/index.md:37]:
```suggestion
This manifest member is used by app stores and catalogs when publishing and listing web apps, so browser compatibility is not applicable. Browsers may parse this information, but it's optional and doesn't affect the core functionality of a web app.
```

**Comment by skyclouds2001** [comment]
Updated!

**Comment by zhowan2020** [comment]
[](url)

**Comment by zhowan2020** [comment]
Duplicate of #`

**Review by dipikabh — APPROVED** [review]
Thanks!

**Pull request merged**